src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -36,18 +36,18 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
     }
 
     /**
-     * Parses a {@code String name} into a {@code Name}.
+     * Parses a {@code String name} into a {@code Company}.
```

Comment:
Maybe the parameter name here can be changed to company to keep things consistent and clear! I understand where youre coming from though i think theres nothing wrong with this either since we are parsing the company name.